packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -484,6 +484,8 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
         view
         returns (bytes memory)
     {
+        // TODO: Consider adding `chainId` and `relayNonce` to establish relay uniqueness, and removing unneccessary
```

Comment:
I think in a future PR we can remove a lot of these params don't establish {deposit, relay} uniqueness.